plainly-plugin/src/node/request.ts

```diff
@@ -69,4 +91,19 @@ async function get(path: string, apiKey: string) {
   return request(getOptions);
 }
 
-export { get };
+async function post(path: string, apiKey: string, body: FormData | string) {
```

Comment:
can we simply have two posts: `post` and `postFormData` and have no typeof then, it's also more readble in the code itself then